### PR TITLE
CXP-372: add Connection fixtures

### PR DESCRIPTION
**Pull request by pchasle**
**Description (for Contributor and Core Developer)**

- [x] create a role for source connections:
  - [ ] give it all Web API permissions
- [x] create a role for destination connections:
  - [ ] give it only "List" Web API permissions
- [x] create a group for SAP connection:
  - [ ] add the group in Master catalog category tree in the “Allowed to own products”
- [x] create a group for Alkemics connection:
  - [ ] add the group in Master catalog category tree in the “Allowed to own products”
- [x] create a group for Translations.com connection:
  - [ ] add the group in Master catalog category tree in the “Allowed to own products”
- [x] create a group for Magento connection:
  - [ ] add the group in Master catalog category tree in the “Allowed to view products”
- [x] create a destination connection : Magento
- [x] create 2 source connections: SAP and Alkemics
- [x] create a connection with the other type: Translations.com
- [x] Add logo

**Definition Of Done (for Core Developer only)**

| Q                                 | A
| --------------------------------- | ---
| Added Specs                       | Todo
| Added legacy Behats               | Todo
| Added acceptance tests            | Todo
| Added integration tests           | Todo
| Changelog updated                 | Todo
| Review and 2 GTM                  | Todo
| Micro Demo to the PO (Story only) | Todo
| Migration script                  | -
| Tech Doc                          | -

`Todo`: Pending / Work in progress
`OK`: Done / Validated
`-`: Not needed

### CODE REVIEW

File: src/Akeneo/Connectivity/Connection/back/Infrastructure/Install/FixturesLoader.php [hunk 1]
```diff
@@ -0,0 +1,288 @@
...
+    }
+
+    private function uploadImage(string $file): FileInfoInterface
+    {
+        $rawFile = new \SplFileInfo(__DIR__ . '/../Symfony/Resources/fixtures/images/' . $file);
+
+        return $this->fileStorer->store($rawFile, FileStorage::CATALOG_STORAGE_ALIAS);
+    }
+
+    private function validate(object $object): void
```
Comment (tseho) [inline src/Akeneo/Connectivity/Connection/back/Infrastructure/Install/FixturesLoader.php:269]: are you sure the type `object` exists in PHP ?

Comment (pchasle) [reply src/Akeneo/Connectivity/Connection/back/Infrastructure/Install/FixturesLoader.php:269]: Yes, since php 7.2 :)

Comment (pierrejolly) [reply src/Akeneo/Connectivity/Connection/back/Infrastructure/Install/FixturesLoader.php:269]: Yes since PHP 7.2 I think

Comment (tseho) [reply src/Akeneo/Connectivity/Connection/back/Infrastructure/Install/FixturesLoader.php:269]: yes, my bad

**Pull request merged**